### DISCUSSION TITLE
Does CustomTkinter supports Linux distributions like Ubuntu ?

### DISCUSSION THREAD

**Discussion by learnerAbhishek** [TomSchimansky/CustomTkinter#699] [Q&A]
Hi. I just ran the sample code provided in the examples and the results in Ubuntu are somewhat distorted. 

Adding some attachments.

![Screenshot from 2022-12-01 00-19-46](https://user-images.githubusercontent.com/17941623/204883728-fae75d0c-c8ac-4438-8882-fb663815f78c.png)
![Screenshot from 2022-12-01 00-21-12](https://user-images.githubusercontent.com/17941623/204883877-5bb38c15-77ed-4fde-9547-1d89e713848f.png)

My ubuntu distribution has the following configuration.
Linux ubuntu 5.15.0-53-generic #59-Ubuntu SMP Mon Oct 17 18:53:30 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by Akascape** [comment]
@learnerAbhishek 
Here is the issue first reported; https://github.com/TomSchimansky/CustomTkinter/issues/206
But I am not facing any issue in Ubuntu 22.

**Reply by learnerAbhishek** [reply]
Hi, @Akascape I followed the approaches mentioned in #206. Unfortunately, it hasn't worked. I tried the following solutions

1 Creating a fonts folder with 3 files in ~/.local/share.
2 I tried switching from python 3.8 to 3.10 and 3.11.
3 Installed customtkinter through pip.

Please let me know what other things I can try.
Thank you

**Reply by Akascape** [reply]
@learnerAbhishek Update Ubuntu.

**Reply by learnerAbhishek** [reply]
My current Ubuntu Version is 22.04.1 LTS.
I can't upgrade it. Anyways thanks for your reply.

I will post a solution if I can get it working.

**Comment by KarfallaDiaby** [comment]
Salut à tous ! J'ai essayer d'installer customtkinter sur debian 12 mais sans suces. Quelqu'un pourrait m'aider ?

**Comment by NanoParticles** [comment]
My project uses custom tkinter and it has been confirmed to be able to run on linux

**Comment by Anonymous6598** [comment]
The same thing as @NanoParticles had but not suggested. Gui will be horrible (window widgets).

**Reply by NanoParticles** [reply]
The widgets was showing fine. There was no visual glitches

**Reply by Anonymous6598** [reply]
Ok, than you can use it normally.
